app/views/articles/new.html.erb

```diff
@@ -0,0 +1,33 @@
+
+<%= form_for :article, url: articles_path do |f| %>
+
+  <% if @article.errors.any? %>
+    <div id="error_explanation">
+      <h2>
+        <%= pluralize(@article.errors.count, "error") %> prohibited
+        this article from being saved:
```

Comment:
a little strange how this was wrapped but whitespace is ignored and this is how it was in the tutorial so I'm going to leave it as is.